HUD/GetMouseEvent.md

```diff
@@ -1,12 +1,12 @@
 ---
 ns: HUD
-aliases: ["0x632B2940C67F4EA9"]
+aliases: ["0x632B2940C67F4EA9", "_GET_SCALEFORM_MOVIE_CURSOR_SELECTION"]
 ---
-## _GET_SCALEFORM_MOVIE_CURSOR_SELECTION
+## GET_MOUSE_EVENT
  
 ```c
 // 0x632B2940C67F4EA9
-BOOL _GET_SCALEFORM_MOVIE_CURSOR_SELECTION(int scaleformHandle, cs_type(Any*) bool* received, cs_type(Any*) int* selectionType, cs_type(Any*) int* context, int* slotIndex);
+BOOL GET_MOUSE_EVENT(int scaleformHandle, cs_type(Any*) bool* received, cs_type(Any*) int* selectionType, cs_type(Any*) int* context, int* slotIndex);
```

Comment:
Descriptions of the params to set:

- eventType -- The specific type of mouse event that has taken place.
- uid -- A distinct identification for the item that initiated the mouse event.
- context -- The context in which this item occurred, which could be a column index, submenu, or a similar reference.